docs/reference/filter_field_definition.rst

```diff
@@ -314,8 +315,8 @@ In this example, ``getWithOpenCommentField`` and ``getWithOpenCommentFilter`` im
                 ->add('author')
                 ->add('with_open_comments', CallbackFilter::class, [
     //                'callback'   => [$this, 'getWithOpenCommentFilter'],
```

Comment:
Can you do the same change on the method getWithOpenCommentFilter ?

Reply:
Of course